packages/openapi-request-validator/index.ts

```diff
@@ -182,6 +191,7 @@ export default class OpenAPIRequestValidator
     let err;
     let schemaError;
     let mediaTypeError;
+    this.logger.debug('validating', request);
```

Comment:
please use https://www.npmjs.com/package/debug for debug log messages and remove these.

Reply:
i'm not using them for debugging, but to track how my app behaves in production, so all the logs (json, produced by a bunyan), from more than one app, are collected and pushed to a central logging server. the debug package wouldn't work for me in this case. I could change these to .info, but we said logging should be opt-in.